src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Windows/MyExtensionsApp.Windows.csproj

```diff
@@ -16,10 +16,12 @@
 		<!-- This bundles the .NET Core libraries (Uncomment for packaged builds)  -->
 		<SelfContained>true</SelfContained>
 		<DefaultLanguage>en</DefaultLanguage>
+		<ApplicationIcon>Images\logo.ico</ApplicationIcon>
```

Comment:
ApplicationIcon isn't required or useful for Windows target as icon etc is defined by package.appxmanifest file